qiskit/providers/basejob.py

```diff
@@ -43,6 +45,18 @@ def backend(self):
         """Return the backend where this job was executed."""
         return self._backend
 
+    def done(self) -> bool:
+        """Return whether the job has successfully run."""
+        return self.status() == JobStatus.DONE
+
+    def running(self):
```

Comment:
Can you add the return type hint here and to `cancelled()`, for consistency?

Reply:
Definitely! That's a nice catch, thanks!